tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -115,10 +115,10 @@ describeSmokeSuite("S300", `Verifying balances consistency`, (context, testIt) =
       specVersion < 2000
         ? apiAt.query.democracy.preimages.entries()
         : ([] as [StorageKey<[H256]>, Option<any>][]),
-      (specVersion >= 1900 && runtimeName == "moonbase") || specVersion >= 2000
+      specVersion >= 2000
```

Comment:
This is wrong. It will fail for RT1900 on moonbase

Reply:
There are no deployed moonbase chains RT1900 or lower, I don't see the value of keeping redundant constraints. My view is that long term support of all current tests being fully backwards compatible is messy to maintain.

If you wanted to check out a previous RT and run tests on it, you always have the option of running the tests in the tag of that release.

Reply:
I don't fully agree, we should be able to run those test on any version of the runtime (excluding the earlier version), so that we can applied them to previous fork and also verify previous consistency when doing troubleshooting. It happened few times in the past where I have to run new smoke test on old RT to find the origin of a broken unknown at the time consistency.